packages/note-access/src/random/makeRand.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-bitwise */
+
+export default function makeRand(seed, size = 4) {
```

Comment:
Unless I'm missing something, this is not used anywhere.

Reply:
removed